tests/unit/20_config.bats

```diff
@@ -145,3 +145,17 @@ teardown() {
   echo "status: "$status
   assert_success
 }
+
+@test "(config) deploy specific DOKKU_DEPLOY_BRANCH" {
+  run ssh dokku@dokku.me config:set --global DOKKU_DEPLOY_BRANCH=global-branch
```

Comment:
What do you mean, its not testing anything new? We test whether the deploy of the `global-branch` succeeds successfully. Is there a better way to do this?